### PR TITLE
Add example in Readme for the Oauth2.0 flow

### PR DESCRIPTION
**Pull request by duykhoa**
This pull request adds an example for the usage of this library in the Oauth 2.0 flow, it is related to the issue #81 .

### CODE REVIEW

**Comment by mtomov** [comment]
https://github.com/googleapis/google-auth-library-ruby/issues/81#issuecomment-329302065

This solution definitely works. What's different are a few additional attributes:

```ruby
secrets = Google::APIClient::ClientSecrets.new(
      {
        "web" =>
          {
            "access_token" => current_user.token,
            "refresh_token" => current_user.refresh_token,
            "client_id" => Rails.application.secrets[:google_client_id],
            "client_secret" => Rails.application.secrets[:google_secret]
          }
      }
```

maybe it's good to clarify if/when those are needed as well.

File: README.md [hunk 1]
```diff
@@ -169,6 +169,23 @@ list_files = drive.list_files()
...
 ```
 
+### Example (Oauth 2.0 Web Server application)
+
+```ruby
+  require 'google/api_client/client_secrets.rb'
+  require "google/apis/oauth2_v2"
+
+  Google::Apis::Oauth2V2::Oauth2Service.new.tap do |client|
+    authorization = Google::APIClient::ClientSecrets.new(
```
Comment (kirrg001) [inline README.md:179]: How to install the client secret package with their new generated package system?
"google-api-client" is deprecated.

Comment (dazuma) [reply README.md:179]: It's been moved to google-apis-core.

**Comment by bajajneha27** [comment]
Hi, thank you for raising this PR and apologies for long delay delay in responding. At the moment, we won't be able to address this but we're definitely going to work on it. I've created another issue to keep track of it, you can follow it https://github.com/googleapis/google-auth-library-ruby/issues/367
I'm closing this PR for now. Please feel free to re-open it in case you have any other concern.

**Pull request closed without merging**